src/VisualStudio/Setup.Dependencies/AssemblyRedirects.cs

```diff
@@ -1,37 +1,11 @@
 [assembly: ProvideDependencyBindingRedirection("Microsoft.DiaSymReader.PortablePdb.dll")]
 [assembly: ProvideDependencyBindingRedirection("System.Collections.Immutable.dll")]
-//[assembly: ProvideDependencyBindingRedirection("System.Collections.dll")] // no implementation assembly for net46
-//[assembly: ProvideDependencyBindingRedirection("System.Collections.Concurrent.dll")] // no implementation assembly for net46
-//[assembly: ProvideDependencyBindingRedirection("System.Diagnostics.Contracts.dll")] // no implementation assembly for net46
-//[assembly: ProvideDependencyBindingRedirection("System.Diagnostics.Tools.dll")] // no implementation assembly for net46
-[assembly: ProvideDependencyBindingRedirection("System.Diagnostics.FileVersionInfo.dll")]
-//[assembly: ProvideDependencyBindingRedirection("System.IO.dll")] // no implementation assembly for net46
-[assembly: ProvideDependencyBindingRedirection("System.IO.Compression.dll")]
-[assembly: ProvideDependencyBindingRedirection("System.IO.FileSystem.dll")]
-[assembly: ProvideDependencyBindingRedirection("System.IO.FileSystem.Primitives.dll")]
-//[assembly: ProvideDependencyBindingRedirection("System.Linq.Expressions.dll")] // no implementation assembly for net46
-//[assembly: ProvideDependencyBindingRedirection("System.Linq.Parallel.dll")] // no implementation assembly for net46
-//[assembly: ProvideDependencyBindingRedirection("System.ObjectModel.dll")] // no implementation assembly for net46
 [assembly: ProvideDependencyBindingRedirection("System.Reflection.Metadata.dll")]
-[assembly: ProvideDependencyBindingRedirection("System.Runtime.InteropServices.RuntimeInformation.dll")]
-//[assembly: ProvideDependencyBindingRedirection("System.Runtime.Numerics.dll")] // no implementation assembly for net46
-[assembly: ProvideDependencyBindingRedirection("System.Security.Cryptography.Algorithms.dll")]
-[assembly: ProvideDependencyBindingRedirection("System.Security.Cryptography.Encoding.dll")]
-[assembly: ProvideDependencyBindingRedirection("System.Security.Cryptography.Primitives.dll")]
-[assembly: ProvideDependencyBindingRedirection("System.Security.Cryptography.X509Certificates.dll")]
-//[assembly: ProvideDependencyBindingRedirection("System.Text.Encoding.dll")] // no implementation assembly for net46
-//[assembly: ProvideDependencyBindingRedirection("System.Text.Encoding.Extensions.dll")] // no implementation assembly for net46
-//[assembly: ProvideDependencyBindingRedirection("System.Text.RegularExpressions.dll")] // no implementation assembly for net46
 [assembly: ProvideDependencyBindingRedirection("System.Threading.Tasks.Extensions.dll")]
-//[assembly: ProvideDependencyBindingRedirection("System.Threading.Tasks.Parallel.dll")] // no implementation assembly for net46
 [assembly: ProvideDependencyBindingRedirection("System.Text.Encoding.CodePages.dll")]
-[assembly: ProvideDependencyBindingRedirection("System.ValueTuple.dll")]
```

Comment:
Commented elsewhere too: not sure why you're removing this and re-adding a pkgdef doing it manually. I'm guessing it's either something subtly different with the redirect produced or "that way the hack is all in one place". If the latter, it seems we're just confusing the situation by this file actually "missing" something.